packages/inter-protocol/src/psm/psm.js

```diff
@@ -118,6 +118,9 @@ export const start = async (zcf, privateArgs, baggage) => {
   const anchorPool = provideEmptyZcfSeat('anchorPoolSeat');
   const feePool = provideEmptyZcfSeat('feePoolSeat');
   const stage = provideEmptyZcfSeat('stageSeat');
+
+  let mintedOutstanding = AmountMath.makeEmpty(stableBrand);
```

Comment:
not clear to me what this value is for. Making `burnLosses` throw an error if `toBurn` is greater than what we've yet minted?

Shouldn't `burnMinted` take the exact same amount as `mintMinted` ? 

As is, I'm not sure this value and those two new functions are in improvement.